launch/tier4_localization_launch/launch/localization.launch.xml

```diff
@@ -6,10 +6,13 @@
   <arg name="random_downsample_filter_param_path"/>
   <arg name="ndt_scan_matcher_param_path"/>
   <arg name="localization_error_monitor_param_path"/>
+  <arg name="eagleye_param_path"/>
 
   <arg name="input/pointcloud" default="/sensing/lidar/top/outlier_filtered/pointcloud" description="The topic will be used in the localization util module"/>
   <arg name="use_pointcloud_container" default="true" description="launch pointcloud container"/>
   <arg name="pointcloud_container_name" default="pointcloud_container"/>
+  <arg name="localization_mode" default="lidar" description="select localization mode. lidar, gnss"/>
```

Comment:
Since current localization architecture assumes that its components belongs to either `pose_estimator` or `twist_estimator`, IMHO it is better to rename this parameter name to something like as follows:
```suggestion
  <arg name="pose_estimator_mode" default="lidar" description="select localization mode. lidar, gnss"/>
```

Reply:
Fixed.